sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -436,11 +436,12 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
-## Known Gaps from Previous Library
+## Plugins
+ 
+In the previous library, `Microsoft.Azure.ServiceBus`, users could [register plugins](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.ServiceBus/src/QueueClient.cs#L527) that would alter an outgoing message before serialization, or alter an incoming message after being deserialized. These extension points allowed users of the Service Bus library to use common OSS extensions to enhance their applications without having to implement their own logic, and without having to wait for the SDK to explicitly support the needed feature. For instance, one use of the plugin functionality is to implement the [claim-check pattern](https://www.nuget.org/packages/ServiceBus.AttachmentPlugin/) to send and receive messages that exceed the Service Bus message size limits. 
```

Comment:
Should we perhaps use the link to the API reference for the register plugin API instead of pointing to source code which is subject to changes?